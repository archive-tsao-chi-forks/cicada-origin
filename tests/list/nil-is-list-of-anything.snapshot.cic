@show
@the List(Nat)
nil

@show
@the List(List(Nat))
nil
