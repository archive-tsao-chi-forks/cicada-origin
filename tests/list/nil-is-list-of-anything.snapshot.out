@the List(Nat) nil
@the List(Nat) nil
@the List(List(Nat)) nil
@the List(List(Nat)) nil

